Assets/Scripts/Player/PlayerMovementController.cs

```diff
@@ -28,8 +28,8 @@ void Start()
     // Update is called once per frame
     void Update()
     {
-        float moveHorizontal = Input.GetAxis("Horizontal");
-        float moveVertical = Input.GetAxis("Vertical");
+        float moveHorizontal = Input.GetKey(KeyCode.D) ? 1 : (Input.GetKey(KeyCode.A) ? -1 : 0);
```

Comment:
No, let's keep Input.GetAxis, it's a good practice and also allows for minute movement if necessary